eapp/config/app-config.js

```diff
@@ -1,5 +1,5 @@
-exports.clientId = ''
-exports.clientSecret = ''
+exports.clientId = 'f41a0704070ceb1e1064'
```

Comment:
You should not commit app-config.js file with your clientId and secretKey.  
Please revert back to the original file and you need to change your clientId and SecretKey now as it is now in Github repository. We should remove app-config.js from git index so it is no longer tracked in git. Even though it is in .gitignore, once it is committed to git, .gitignore file has no effect.